bin/runtime/src/lib.rs

```diff
@@ -470,6 +494,8 @@ pub const TREASURY_BURN: u32 = 0;
 pub const TREASURY_PROPOSAL_BOND: u32 = 0;
 // The proposer should deposit max{`TREASURY_PROPOSAL_BOND`% of the proposal value, $10}.
 pub const TREASURY_MINIMUM_BOND: Balance = 1000 * CENTS;
+// NOTE: I pulled this value from the air
```

Comment:
Maybe add here that this value needs to be understood and possibly adjusted before landing on {test,main}net? Probably with some caps and exclamation marks ;)

Reply:
done